### PR TITLE
critest: remove dependency of source code

### PR DESCRIPTION
**Pull request by feiskyer**
critest requires Go and cri-tools source code to build e2e.test and benchmark.test. This makes critest hard to install and need to switch source code branches when running old versions.

This PR removes this requirement and makes only a single `critest` binary required for CRI validation tests. After this PR, downloading from a release is encouraged to install critest.

It will also help to add validation tests for windows containers #223 .

/cc @Random-Liu @mrunalp @runcom @resouer

### CODE REVIEW

File: docs/validation.md [hunk 1]
```diff
@@ -6,15 +6,20 @@ CRI validation testing is currently Alpha and still under quick iterations. We e
 
 ## Install
 
-The test suites can be installed easily via `go get` command:
+The benchmarking tests binary `critest` can be downloaded from [Releasing page](https://github.com/kubernetes-incubator/cri-tools/releases):
 
 ```sh
-go get github.com/kubernetes-incubator/cri-tools/cmd/critest
+wget https://github.com/kubernetes-incubator/cri-tools/releases/download/v1.0.0-beta.0/critest-v1.0.0-beta.0-linux-amd64.tar.gz
```
Comment (feiskyer) [inline docs/validation.md]: Note that the file does not exist yet, it should be uploaded after beta version released.

**Comment by Random-Liu** [comment]
This is what I want. I just don't know whether we have time to finish this, so I didn't file the issue.
Thanks!

**Comment by feiskyer** [comment]
ping @Random-Liu

**Comment by umohnani8** [comment]
Tested it and worked for me! LGTM

**Comment by feiskyer** [comment]
Rebased to master branch

**Comment by Random-Liu** [comment]
@feiskyer The parallel mode doesn't work. Only 8 test runs after this change.
```
Random Seed: 1521084457 - Will randomize all specs
Parallel test node 1/8.
...
Ran 8 of 8 Specs in 9.863 seconds
SUCCESS! -- 8 Passed | 0 Failed | 0 Pending | 0 Skipped PASS
```

**Pull request merged**